hoomd/hpmc/integrate.py

```diff
@@ -206,12 +206,14 @@ def map_energies(self):
         Note:
             :py:meth:`map_energies` does not support MPI parallel simulations.
 
+        Attention:
+            `map_energies` is not yet implemented in HOOMD v3.x.
+
         Example:
             mc = hpmc.integrate.shape(...)
             mc.shape_param.set(...)
             energy_map = np.asarray(mc.map_energies())
         """
-
```

Comment:
I think we need a pass here?

Reply:
I added a `NotImplementedError` exception.